app/routes/docs/styleguide.md

```diff
@@ -0,0 +1,65 @@
+---
+meta:
+  title: GCN - Circulars Style Guide
+---
+
+## Style Guide
```

Comment:
How about an example circular?

Reply:
Annoyingly the character spacing length of circulars is longer than that of the webpage inset functionality, so it has seemingly random line spacing. We can modify the circular text so it looks nice, or edit the page width